components/tikv_util/src/lib.rs

```diff
@@ -312,40 +311,41 @@ impl<T: FnOnce()> Drop for DeferContext<T> {
     }
 }
 
-/// Represents a value of one of two possible types (a more generic Result.)
-#[derive(Debug, Clone)]
-pub enum Either<L, R> {
-    Left(L),
-    Right(R),
+/// Provide more convenient functions for `futures::future::Either`
+pub trait EitherExt<L, R> {
```

Comment:
I suggest to implement `From` trait instead of use future's version directly. Although they share similar layout, but `Either` from `tikv_util` has more general design that doesn't have to be related to future.

Reply:
Done